_std/macros/qdel.dm

```diff
@@ -1,2 +1,8 @@
 ///if thing is null or has been qdeled
 #define QDELETED(thing) (isnull(thing) || thing.disposed)
+
+/// qdel & null an item out
+#define QDEL_NULL(item) qdel(item); item = null
+
+/// qdel every item in a list, then cut the list
+#define QDEL_LIST(L) if(L) { for(var/I in L) qdel(I); L.Cut(); }
```

Comment:
```suggestion
#define QDEL_LIST(L) if(L) { for(var/I as anything in L) qdel(I); L.Cut(); }
```
Dunno if it matters in this case given lack of type?

Reply:
It's equivalent yeah